LandSandBoat/server
# Fix race specific equipment and sexchange issue

**_I affirm:_**

## What does this pull request do?
This PR prevents players that use the sexchange addon from equipping gear they should not be able to equip (such as RSE gear of different races). This is done by adding a mod that stores the valid races for any RSE gear and then checking this mod server side when players change gear (thus preventing the change). The mod stores a multi-bit flag similar to how the jobs flag works.

Fixes https://github.com/LandSandBoat/server/issues/1147

## Steps to test these changes
Try to equip different RSE gear after using the sexchange addon to change your race on the client side. Notice that you are no longer able to equip RSE gear of the race you changed to.
